### ISSUE TITLE
1.7.6.5 very unstable and slow

### ISSUE THREAD

**Issue by 01park** [PrestaShop/PrestaShop#18772]
After updating to 1.7.6.5 our site became very unstable and slow.

I need to reset Apache 4 to get it working again and after 5-6 minutes it repeats again being slow

We get the message time out, server can not be reached. 

I checked, but there is errors coming and when I turn on debug, also nothing happens. 

* PrestaShop version: 1.7.6.5
* PHP version: 7.2.29

**Comment by florine2623** [comment]
Hi @01park ,
Could you try to comment module gamification ?
In folder /modules, ```gamification``` becomes ```_gamification```
Please check and let me know !

**Comment by 01park** [comment]
Hi, 

I attached the screenshot of what I changed. Is this what you mean?

<img width="492" alt="Screenshot 2020-04-23 at 11 15 12" src="https://user-images.githubusercontent.com/46492443/80088097-f5bff800-8553-11ea-8906-d017b9327386.png">

**Comment by florine2623** [comment]
Yes that's it !

**Comment by 01park** [comment]
I tried, but still need to reset Apache

**Comment by florine2623** [comment]
Here are a few things to try and to check : 
- clear all cache both manually and from your FTP
- regarding Smarty cache, enable only if a modification has been done
- activate the cache
- deactivate all overrides and module not developed by PrestaShop
- check in Positions if you have hooked modules that are not used. These might send queries in the background hence the latency

**Comment by 01park** [comment]
Hi,

I tried all of these and then gave up. I though maybe there is something wrong with the server and decided to transfer everything to a new server.

All went good until I wanted to restore the database its giving me this error

Error: SyntaxError: Unexpected token < in JSON at position 0

Since I'm getting this error I though, maybe this is causing the site also to respond slow. 

I turned on debug mode, but no error coming. Do you know where I need to search for this issue?

**Comment by 01park** [comment]
Today, for some reason the speed has comeback. Direct after the update I noticed a large amount of crawlers on the site, may this caused the performance to go down.

Does anyone know how to fix the JSON issue with the Database?

**Comment by 01park** [comment]
After the speed on the site came back and I made a new backup, I was able to restore. The site on the new server.

This can be marketed als solved.

If anyone has time, can they tell me why a this error with JSON came?

**Comment by florine2623** [comment]
Hi @01park 
Good to hear that the speed has come back.
Please open a new issue regarding the JSON error. Do not forget to explain the exact steps to reproduce your error, as well as screenshots.
Thanks!

**Comment by 01park** [comment]
Unfortunately the problem did not stay away for good and returned. 
I have been doing some testing and problem shooting.

At first, I though it might be the server. I build a new PS on a new server and migrated the data from the poor performance store to the new one. Once the data was transferred. Everything look fine and I though great problem solved.

As I'm using multistore, I put all stores in to maintenance because I needed to change IP address to move everything to the new server. Once I did this, the Back Office once again became super fast and stable. I thought, what a coincidence.

Once the IP address moved to the new server and I changed the domain of the newly build store to my main store. All of sudden the same performance issue started again. I now realised that I has to be something. 

Today I moved back to my original store and server and left 95% of my multistores in maintenance. I can work in Back office and two store remain open, but once I put the main store online, it crashes again. 

I checked the amount of visitors and find it higher than normal, but this should not be a problem as it was not a problem before I did this update. 

I honestly don't know where to look for the answer of this problem, all help is welcome
